### PR TITLE
C10938

### PR DESCRIPTION
**Pull request by KyryloBrener**

### CODE REVIEW

File: cypress/integration/invoices/settings/invoices.settings.checkSystemBatchGroup.spec.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import TestType from '../../../support/dictionary/testTypes';
+
+describe('ui-invoices-settings: System Batch Group deletion', () => {
+  const batchGroup = { ...NewBatchGroup.defaultUiBatchGroup };
+  const systemBatchGroup = { ...NewBatchGroup.defaultUiBatchGroup };
+  const systemBatchGroupName = 'FOLIO';
+  const systemBatchGroupDescription = 'System default';
+  before(() => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.visit(`${TopMenu.settingsInvoice}${SettingsInvoices.settingsInvoicePath.batchGroups}`);
```
Comment (dmitriyfolio) [inline cypress/integration/invoices/settings/invoices.settings.checkSystemBatchGroup.spec.js:13]: let's sync with @IhorBohdan  and me about settings menu paths

File: cypress/integration/invoices/settings/invoices.settings.checkSystemBatchGroup.spec.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  });
+
+  it('C10938 FOLIO Batch group is created by system and can only be edited', { tags: [TestType.smoke] }, () => {
+    systemBatchGroup.name = systemBatchGroupName;
+    systemBatchGroup.description = systemBatchGroupDescription;
+    SettingsInvoices.waitBatchGroupsLoading();
+    SettingsInvoices.checkThatSystemBatchGroupCantBeDeleted(systemBatchGroupName);
```
Comment (dmitriyfolio) [inline cypress/integration/invoices/settings/invoices.settings.checkSystemBatchGroup.spec.js:20]: difficult naming. Change to checkNotDeletingGroup(). sysBatchGroupName will help to determine the reason.

File: cypress/support/fragments/invoices/settingsInvoices.js [hunk 3]
```diff
@@ -33,33 +33,54 @@ export default {
...
-
-    cy.expect(EditableListRow({ index: rowNumber })
-      .find(MultiColumnListCell({ columnIndex: 2 }))
-      .has({ content: createdByAdmin }));
+  checkBatchGroup: (batchGroup) => {
+    cy.do(MultiColumnListCell({ content: batchGroup.name }).perform(
+      element => {
+        const rowNumber = element.parentElement.parentElement.getAttribute('data-row-index');
+        const createdByAdmin = `${DateTools.getFormattedDateWithSlashes({ date: new Date() })} by ADMINISTRATOR, DIKU`;
+        cy.expect(EditableListRow({ index: +rowNumber.split('-')[1] })
```
Comment (dmitriyfolio) [inline cypress/support/fragments/invoices/settingsInvoices.js:51]: add const with EditableListRow({ index: +rowNumber.split('-')[1] })

File: cypress/integration/invoices/settings/invoices.settings.checkSystemBatchGroup.spec.js (hunk 2 above)
Comment (KyryloBrener) [reply cypress/integration/invoices/settings/invoices.settings.checkSystemBatchGroup.spec.js]: Updated

File: cypress/support/fragments/invoices/settingsInvoices.js (hunk 3 above)
Comment (KyryloBrener) [reply cypress/support/fragments/invoices/settingsInvoices.js]: Updated

**Review by dmitriyfolio — APPROVED** [review]

**Review by MaksimTerekhov — APPROVED** [review]

**Pull request merged**